src/components/Choropleth.vue

```diff
@@ -150,59 +94,54 @@ const tooltipForFeature = (feature: Feature) => {
     return `<div><strong>${name}</strong></div><div>${indicatorValues}</div>`;
 };
 
-const createTooltips = {
-    onEachFeature: (feature: Feature, layer: Layer) => {
-        layer.bindTooltip(tooltipForFeature(feature)).openTooltip();
-        layer.on({
-            click: async () => {
-                const country = feature.properties[featureProperties.country];
-                // select feature's country, or unselect if click on it when already selected
-                let countryToSelect: string;
-                if (country === selectedCountryId.value) {
-                    countryToSelect = "";
-                } else {
-                    countryToSelect = country;
-                }
-                router.push(`/${APP_BASE_ROUTE}/${selectedIndicator.value}/${countryToSelect}`);
-            }
-        });
-    }
+const createTooltips = (feature: Feature, layer: Layer) => {
```

Comment:
Maybe we should rename this method since it's isn't just creating tooltips any more.

Reply:
Leave it to me on the tooltips branch. I'll split this function into a bit that comes from a useTooltips composable, and a bit that remains here.

Reply:
something like `bindTooltipAndClickEvent`? thats what ive gone with for now

Reply:
..or could be something more generic reflecting what leaflet is doing, like `configureGeojsonLayer`

Reply:
done